public/locales/cs/common.json

```
@@ -38,9 +38,9 @@
     "movapp_goal_title": "<strong>Cíl Movapp.cz</strong>",
```

`<strong>` tuto nič nerobí kedže ten text sa aj tak renderuje vnutri nadpisu, ktorý už je tučne zo svojej podstaty. Všeobecne by bolo lepšie nemať v prekladoch žiadne HTML tagy napriamo. Snažiť sa držať informácie o formátovaní v komponentách kde sa preklad použije, pokiaľ možno. Vymazal by som zatial aspoň tieto zbytočné `<strong>` v nadpisoch. 

```suggestion
    "movapp_goal_title": "Cíl Movapp.cz",
```